### PR TITLE
2468 ui  implement bank branding

### PR DESCRIPTION
**Pull request by AnnaGuzniak**
viber_image_2019-12-23_13-46-49

### CODE REVIEW

**Comment by AnnaGuzniak** [comment]
dropdown items

**Comment by AnnaGuzniak** [comment]
image

**Comment by AnnaGuzniak** [comment]
image

File: lib/spoon-ui.scss [hunk 1]
```diff
@@ -543,8 +542,8 @@ a:hover {
 }
 
 .form-group label {
-  font-size: 18px;
-  line-height: 23px;
+  font-size: 15px;
```
Comment (olikjh) [inline lib/spoon-ui.scss]: I think, it's better to use scss variables for sizes. which are dividible by 4
```
font-size: $unit-size*4;
```

**Pull request merged**